bee-bundle/src/transaction.rs

```diff
@@ -382,6 +429,14 @@ impl Transactions {
     }
 }
 
+/// Milestone - TODO - builder?
+
+#[derive(Debug, Clone)]
+pub struct Milestone {
+    pub hash: Hash,
+    pub index: u32,
```

Comment:
Fields should not be pub but let's let them like that as I think I'll be moving Milestone to the protocol crate soon.